### PR TITLE
feat: add madrid agenda

### PR DESCRIPTION
**Pull request by thulieblack**
solves #187

### CODE REVIEW

File: config/speakers.json [hunk 1]
```diff
@@ -2,6 +2,117 @@
...
+				"type": "Community Talk"
+			},
+			{
+				"time": "5:00 PM CEST - 5:30 PM CEST",
+				"session": "",
+				"type": "Break"
+			},
+			{
+				"time": "5:30 PM CEST - 6:00 PM CEST",
+				"session": "Leveraging AsyncAPI To Detect Anomalies In Smart Ports Platforms Using Model-Driven Techniques: ...",
```
Comment (thulieblack) [inline config/speakers.json:86]:
```suggestion
				"session": "Leveraging AsyncAPI To Detect Anomalies In Smart Ports Platforms Using Model-Driven Techniques,
```

File: config/speakers.json [hunk 2]
```diff
@@ -2,6 +2,117 @@
...
+				"type": "Community Talk"
+			},
+			{
+				"time": "5:00 PM CEST - 5:30 PM CEST",
+				"session": "",
+				"type": "Break"
+			},
+			{
+				"time": "5:30 PM CEST - 6:00 PM CEST",
+				"session": "Leveraging AsyncAPI To Detect Anomalies In Smart Ports Platforms Using Model-Driven Techniques,
```
Comment (thulieblack) [inline config/speakers.json:86]:
```suggestion
				"session": "Leveraging AsyncAPI To Detect Anomalies In Smart Ports Platforms Using Model-Driven Techniques",
```

**Review by ivangsa — APPROVED** [review]
lgtm

**Comment by AceTheCreator** [comment]
LGTM!!!

**Review by AceTheCreator — APPROVED** [review]

**Comment by thulieblack** [comment]
@AceTheCreator @ivangsa any changes needed? before we get this merged?

**Comment by AceTheCreator** [comment]
> @AceTheCreator @ivangsa any changes needed? before we get this merged?

@thulieblack why do we have the home button text changed?

**Comment by thulieblack** [comment]
London Tickets are done, we need to add the Madrid tickets form next  @AceTheCreator

**Comment by AceTheCreator** [comment]
> London Tickets are done, we need to add the Madrid tickets form next @AceTheCreator

Ikr! But I don't think we need to include the following text "Get Free Tickets (Madrid)"

**Comment by thulieblack** [comment]
Alright letes remove it

**Comment by thulieblack** [comment]
Done @AceTheCreator lets get it merged

**Pull request merged**